worlds/tunic/__init__.py

```diff
@@ -241,8 +241,15 @@ def stage_generate_early(cls, multiworld: MultiWorld) -> None:
 
     def create_item(self, name: str, classification: ItemClassification = None) -> TunicItem:
         item_data = item_table[name]
-        # if item_data.combat_ic is None, it'll take item_data.classification instead
+        # evaluate alternate classifications based on options
+        # it'll choose whichever classification isn't None first in this if else tree
         itemclass: ItemClassification = ((item_data.combat_ic if self.options.combat_logic else None)
+                                         or (ItemClassification.progression | ItemClassification.useful
+                                             if name == "Glass Cannon" and self.options.grass_randomizer
+                                             and not self.options.start_with_sword else None)
+                                         or (ItemClassification.progression
```

Comment:
should this be 
```suggestion
                                         or (ItemClassification.progression | ItemClassification.useful
```
?

Reply:
Nah, prog useful doesn't feel right for the ladder storage items imo, mostly since there's a few different ones you can use

Reply:
Hmm maybe it's just the prog useful stuff confusing me again, but if the shield is already considered useful it feels weird to remove that flag and just mark it as progression, especially if we're *not* doing that for glass cannon lol

Reply:
I do think that there's an argument to be made for the general utility of the shield being relevant enough to keep it as a 'Useful' item. It's not a major unblocker, but neither is the Magic Dagger if ice grappling isn't enabled, and yet that's still Progression + Useful, presumably due to its general usefulness.

Reply:
For consistency, we'd also need to make the orb prog useful

Reply:
Is it not already?

Reply:
Actually it probably is lol

Reply:
I just like, it's prog exclusively because of ladder storage here, and there's 5 other item instances that get you ls too (the 4 sword upgrades and orb), so I don't really think it needs the useful tag here

Reply:
Maybe I just don't know what prog useful is actually supposed to mean then.
A progression item that is also useful, or a super solely-progression item that is "more" progressive than other progression items?

Reply:
A progression item that is particularly useful, basically. Like a Sword or something

Reply:
https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/world%20api.md#items

For reference, the description of Useful here

Reply:
We talked on discord, gonna add the useful o: